### PR TITLE
Module 11

### PR DESCRIPTION
**Pull request by HenryRiddallWork**
I had a few issues when accessing the azure site (I think due to me trying to access it using http, then switching to https) but it appears to work for me now. 

Also this includes a random typo fix from last module 😅

### CODE REVIEW

File: .env.j2 [hunk 1]
```diff
@@ -8,7 +8,14 @@ FLASK_DEBUG=true
...
 
+# Required to use HTTPS URLs
+PREFERRED_URL_SCHEME=https
+
 # Required to use MongoDB
 MONGO_CONNECTION_STRING={{ MONGO_CONNECTION_STRING }}
 MONGO_DB_NAME={{ MONGO_DB_NAME }}
-MONBGO_DB_COLLECTION={{ MONBGO_DB_COLLECTION }}
\ No newline at end of file
+MONGO_DB_COLLECTION={{ MONGO_DB_COLLECTION }}
```
Comment (Jonesey13) [inline .env.j2:17]: This is not set correctly on the App Service atm:
image
image

File: README.md [hunk 2]
```diff
@@ -136,6 +136,8 @@ $[1] 127.0.0.1 - - [05/Apr/2024 16:49:39] "GET /static/css/index.css HTTP/1.1" 2
 
 Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser to view the app.
 
+Note: If you are getting "Insecure Transport Error" errors you can add: `OAUTHLIB_INSECURE_TRANSPORT=1` to your .env file for local dev.
```
Comment (Jonesey13) [inline README.md:139]: Probably want this included in your `.env` template file (as it's primarily used for local dev)

File: .env.j2 (hunk 1 above)
Comment (Jonesey13) [reply .env.j2:17]: I've updated the env variable on the app service and it now appears to be working :)

**Review by Jonesey13 — APPROVED** [review]
Looks good to me! Had to fix a small environment variable issue (see below) but it working both on Azure and locally :)

File: .env.j2 (hunk 1 above)
Comment (HenryRiddallWork) [reply .env.j2:17]: Interesting, I wonder how it was working before 😂

**Pull request merged**